apps/health_campaign_field_worker_app/lib/utils/date_conversions.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/foundation.dart';
+import 'package:intl/intl.dart';
+
+class DateConversions {
+  static getFilteredDate(String date, {String? dateFormat}) {
+    if (date.trim().isEmpty) return '';
+    try {
+      var dateTime = DateTime.parse(date).toLocal();
+
+      return DateFormat(dateFormat ?? "dd/MM/yyyy").format(dateTime);
+    } on Exception catch (e) {
+      if (kDebugMode) {
+        print(e);
+      }
+      //return error
+    }
+  }
+
+  static DateTime? getFormattedDateToDateTime(String date) {
+    try {
+      DateFormat inputFormat;
+      inputFormat = date.contains('-')
+          ? DateFormat('dd-MM-yyyy')
```

Comment:
date formate should be passed as params

Reply:
@naveen-egov This accepts input as 'dd-MM-yyyy' or 'dd/MM/yyyy', Its checking which format the input is accordingly returning the dateTime object